src/websocketserver/WebSocketServer.cpp

```diff
@@ -100,7 +100,10 @@ void WebSocketServer::Start()
 	_server.reset();
 
 	websocketpp::lib::error_code errorCode;
-	if (conf->Ipv4Only) {
+	if (conf->ServerHost != "") {
+		blog(LOG_INFO, "[WebSocketServer::Start] Locked to %s", conf->ServerHost);
+		_server.listen(conf->ServerHost, std::to_string(conf->ServerPort), errorCode);
```

Comment:
I'm looking at the websocketpp documentation, and I'm not seeing a matching function for listen(). Do you know which function it's being implicitly cast to? I'm guessing `listen(InternetProtocol const &internet_protocol, uint16_t port, lib::error_code &ec)`

It may be safer in this specific situation to first create an `InternetProtocol` object, then pass it to the listen function.

Reply:
I'm using `listen(	std::string const &host, std::string const &service, lib::error_code &ec)` [docs here](https://docs.websocketpp.org/classwebsocketpp_1_1transport_1_1asio_1_1endpoint.html#a85606665cc9c948194076c2377cb61c0)

As far as I could tell, InternetProtocol doesn't let you bind to a specific address.

Reply:
This feedback thread is still unresolved. What is the path forward here?